src/gluonts/model/rotbaum/_model.py

```diff
@@ -158,8 +178,13 @@ def fit(self, x_train, y_train, max_sample_size=None, seed=1, **kwargs):
         self.model.fit(x_train, y_train, **kwargs)
         y_train_pred = self.model.predict(x_train)
         self.df = pd.DataFrame(
-            {"x": list(x_train), "y_true": y_train, "y_pred": y_train_pred}
-        )
+            {
+                "x": list(np.array(x_train)),  # this uniformly deals with
```

Comment:
```suggestion
                "x": np.array(x_train).tolist(),  # this uniformly deals with
```

AFAIK, you generally want to use `.tolist()`, since it also works for multi-dimensional arrays.

I think using `list(...)` is a bit of a hack, since it just iterates over the array and then collects the elements.

```py
In [5]: np.arange(9).reshape((3, 3)).tolist()
Out[5]: [[0, 1, 2], [3, 4, 5], [6, 7, 8]]

In [6]: list(np.arange(9).reshape((3, 3)))
Out[6]: [array([0, 1, 2]), array([3, 4, 5]), array([6, 7, 8])]
```

Reply:
Actually, it is not necessary to keep track of x, and it just makes the models more bulky. I dropped this altogether.